Makefile

```diff
@@ -197,7 +197,7 @@ endif
 .PHONY: run-tests $(TEST_TARGETS)
 
 docker-build-debug:
```

Comment:
nit: undo. This dockerfile is solely for the e2e tests. It shouldn't be the image we push and use for operators on mainnet.

Reply:
Wouldn't we want to run our tests in an environment as close to production as possible?
I can revert the change and create a dedicated Dockerfile. I have one somewhere. But what pitfalls do you see of us using the same image?